src/locales/en/error.json

```diff
@@ -9,6 +9,7 @@
   "onlyLetter": "Field should contain only letters",
   "emailLength": "Field should contain from 8 to 60 characters",
   "wrongFormat": "Wrong format",
+  "text": "Field should contain from 2 to 700 characters",
```

Comment:
The naming of the constants and object keys should be informative, "text" is not. Change it to describe what is behind this translation.

Reply:
I fixed this.